src/components/ComponentGallery/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Link as GatsbyLink } from "gatsby";
+import "./index.css";
+import pagesData from "../../../public/icds-pages-data.json";
```

Comment:
I think you might have added back the import my commit removed, be worth checking if this has happened elsewhere